# Indexation du g√©nome

#/home/Tools/STAR-STAR_2.4.2a/bin/Linux_x86_64/STAR --runThreadN 30 --runMode genomeGenerate --genomeDir /storage/data/rnaseq/nematodes/incognita/asexevol/asexevol_UNMERGED_CLEAN/STAR_index --genomeFastaFiles /home/data/genome/Meloidogyne_incognita_V3_20131230.fna --sjdbGTFfile /home/data/gff3/MiV3.GTF --sjdbOverhang 75 --genomeChrBinNbits 14 --genomeSAindexNbases 13

# Mapping

/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn W_mito/prinseq/W1_S4_001_clean2.fastq-good_1.fastq W_mito/prinseq/W1_S4_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix W_mito/STAR/W1_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn W_mito/prinseq/W2_S5_001_clean2.fastq-good_1.fastq W_mito/prinseq/W2_S5_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix W_mito/STAR/W2_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn W_mito/prinseq/W3_S6_001_clean2.fastq-good_1.fastq W_mito/prinseq/W3_S6_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix W_mito/STAR/W3_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic

/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J2_mito/prinseq/J2-1_S1_001_clean2.fastq-good_1.fastq J2_mito/prinseq/J2-1_S1_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J2_mito/STAR/J2-1_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J2_mito/prinseq/J2-2_S2_001_clean2.fastq-good_1.fastq J2_mito/prinseq/J2-2_S2_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J2_mito/STAR/J2-2_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J2_mito/prinseq/J2-3_S3_001_clean2.fastq-good_1.fastq J2_mito/prinseq/J2-3_S3_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J2_mito/STAR/J2-3_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic

#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J3_mito/prinseq/J3-1_S4_001_clean2.fastq-good_1.fastq J3_mito/prinseq/J3-1_S4_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J3_mito/STAR/J3-1_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J3_mito/prinseq/J3-2_S5_001_clean2.fastq-good_1.fastq J3_mito/prinseq/J3-2_S5_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J3_mito/STAR/J3-2_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn J3_mito/prinseq/J3-3_S6_001_clean2.fastq-good_1.fastq J3_mito/prinseq/J3-3_S6_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix J3_mito/STAR/J3-3_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic

#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn Fem_mito/prinseq/Fem1_S1_001_clean2.fastq-good_1.fastq Fem_mito/prinseq/Fem1_S1_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix Fem_mito/STAR/Fem1_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn Fem_mito/prinseq/Fem2_S2_001_clean2.fastq-good_1.fastq Fem_mito/prinseq/Fem2_S2_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix Fem_mito/STAR/Fem2_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic
#/home/Tools/STAR-2.5/bin/Linux_x86_64/STAR --runThreadN 30 --runMode alignReads --genomeDir STAR_index/ --readFilesIn Fem_mito/prinseq/Fem3_S3_001_clean2.fastq-good_1.fastq Fem_mito/prinseq/Fem3_S3_001_clean2.fastq-good_2.fastq --sjdbOverhang 75 --outFileNamePrefix Fem_mito/STAR/Fem3_ --outReadsUnmapped Fastx --outMultimapperOrder Random --chimSegmentMin 20 --quantMode TranscriptomeSAM GeneCounts --twopassMode Basic

